TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -460,8 +461,13 @@ public class RoutingManager
                         continue;
                     }
 
-                    bool start = (bool)Constants.ServiceFactory.NetService.IsStartNode(segId, nextNodeId);
-                    ExtSegmentEnd segEnd = segEndMan.ExtSegmentEnds[segEndMan.GetIndex(segId, start)];
+                    bool? start = Constants.ServiceFactory.NetService.IsStartNode(segId, nextNodeId);
+                    if (!start.HasValue) {
+                        Log.Error($"Segment with id: {segId} is not connected to the node {nextNodeId}");
+                        Debug.LogError($"TM:PE RecalculateLaneRoutings - Segment with id {segId} is not connected to the node {nextNodeId}");
```

Comment:
why this one is debug.logerror ?
Log.Error does print to game file but Debug.LogError does not write to TMPE.

Reply:
Hmm interesting, I'll check, I remember that Log.Error didn't write to the game log

Reply:
That's what I thought, `Log.Error` does not write to the game log (TMPE logger only writes to the TMPE.log).